quattor/aii/opennebula/rpms.pan

```diff
@@ -0,0 +1,22 @@
+# #
+# Software subject to following license(s):
+#   Apache 2 License (http://www.opensource.org/licenses/apache2.0)
+#   Copyright (c) Responsible Organization
+#
+
+# #
+# Current developer(s):
+#   Alvaro Simon Garcia <Alvaro.SimonGarcia@UGent.be>
+#   Stijn De Weirdt <Stijn.DeWeirdt@UGent.be>
+#
+
+# 
+# #
+# aii-opennebula, 16.2.0-rc4-SNAPSHOT, rc4_SNAPSHOT20160219150415, 2016-02-19T15:04:15Z
+#
+
+# Template adding aii-opennebula rpm to the configuration
+
+unique template quattor/aii/opennebula/rpms;
+
+"/software/packages"=pkg_repl("aii-opennebula","16.2.0-rc4_SNAPSHOT20160219150415","noarch");
```

Comment:
it is in master. mvn_pack gives:
[INFO] Reactor Summary:
[INFO] 
[INFO] AII server part .................................... SUCCESS [  2.987 s]
[INFO] AII DHCP config generator .......................... SUCCESS [  1.007 s]
[INFO] AII Kickstart generator ............................ SUCCESS [ 32.427 s]
[INFO] AII PXE-linux generator ............................ SUCCESS [ 24.586 s]
[INFO] Opennebula hook for AII ............................ SUCCESS [ 16.325 s]
[INFO] AII ................................................ SUCCESS [  0.077 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS